inductiva/molecules/protein_solvation/protein_solvation.py

```diff
@@ -80,7 +67,7 @@ def simulate(
             n_steps_min: Number of steps for energy minimization.
             run_async: Whether to run the simulation asynchronously.
         """
-        simulator.override_api_method_prefix("proteinsolvation")
+        simulator.override_api_method_prefix("protein_solvation")
```

Comment:
This should be the same as in the output_consts, right?

Reply:
It is, yes!

Reply:
got it 👍